compiler/record-minmax/CMakeLists.txt

```diff
@@ -19,8 +22,16 @@ add_executable(record-minmax ${DRIVER} ${SOURCES})
 target_include_directories(record-minmax PRIVATE include)
 target_include_directories(record-minmax PRIVATE ${Boost_INCLUDE_DIRS})
 target_include_directories(record-minmax PRIVATE ${HDF5_INCLUDE_DIRS})
-
-target_link_libraries(record-minmax ${HDF5_CXX_LIBRARIES})
+unset(HDF5_STATIC_LIBRARIES)
+foreach(LIB ${HDF5_CXX_LIBRARIES})
+  if(LIB MATCHES libsz.so OR LIB MATCHES libz.so)
+    string(REGEX REPLACE "[.]so$" ".a" LIB ${LIB})
+    endif()
+    list(APPEND HDF5_STATIC_LIBRARIES ${LIB})
+endforeach()
+find_library(LIB_AEC NAMES "libaec.a")
```

Comment:
@mhs4670go `find_library` will set `LIB_AEC` cache variable. Maybe you need to unset `LIB_AEC` before `find_library`.